src/main/resources/liquibase/crop_changelog/21_1_0.xml

```diff
@@ -22,4 +22,64 @@
 		</sql>
 	</changeSet>
 
+	<changeSet author="mderamos" id="v21.1.0-2">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="program_location_default"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create program_location_default table
+		</comment>
+		<createTable tableName="program_location_default">
+			<column name="id" type="int" autoIncrement="true">
+				<constraints primaryKey="true"/>
+			</column>
+			<column name="program_uuid" type="varchar(255)">
+				<constraints nullable="false" unique="true"/>
+			</column>
+			<column name="location_id" type="INT(11)">
+				<constraints nullable="false" foreignKeyName="fk_program_location_default" references="location(locid)"/>
+			</column>
+			<column name="created_date" type="timestamp" defaultValueComputed="CURRENT_TIMESTAMP">
+				<constraints nullable="false"/>
+			</column>
+			<column name="created_by" type="int" defaultValue="0">
+				<constraints nullable="false" />
+			</column>
+			<column name="modified_date" type="timestamp">
+				<constraints nullable="true"/>
+			</column>
+			<column name="modified_by" type="int">
+				<constraints nullable="true" />
+			</column>
+		</createTable>
+	</changeSet>
+
+	<changeSet author="mderamos" id="v21.1.0-3">
+		<preConditions onFail="MARK_RAN">
+			<and>
+				<tableExists tableName="program_location_default"/>
+				<not>
+					<sqlCheck expectedResult="0">
+						SELECT COUNT(1) FROM workbench.workbench_project project
+						INNER JOIN workbench.workbench_crop crop
+						WHERE crop.crop_name = project.crop_type
+						AND crop.db_name = DATABASE()
+					</sqlCheck>
+				</not>
+			</and>
+		</preConditions>
+		<comment>
+			Populate program_location_default for existing programs, set 0 as default
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			INSERT INTO program_location_default(program_uuid, location_id)
+			SELECT project.project_uuid, 0 FROM workbench.workbench_project project
```

Comment:
Hi @clarysabel , I just noticed that it's not a deletable location since its ltype = 405. There's a validation where the system prevents the users from deleting locations with ltype=405.

But as a safety precaution, I also added a liquibase script that checks if the location with labbr UKN is existing. If not, reinsert it in the table and use the reinserted value as the default for existing programs.